kala.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/python
 
+import datetime
 import os
 import json
+import uuid
 
 import bottle
 from bottle_mongo import MongoPlugin
 
-
```

Comment:
PEP8 does say 2 spaces at top level (or at least PyCharm whinges about it constantly)